ui/admin/tests/helpers/selectors.js

```diff
@@ -18,6 +18,8 @@ export const FIELD_DESCRIPTION_VALUE = 'description';
 export const FIELD_PASSWORD = '[name=password]';
 export const FIELD_PASSWORD_VALUE = 'password';
 
+export const FIELD_IDENTIFICATION = '[name=identification]';
```

Comment:
It is debatable if this selector should live here. It's use only within authentication test, but since authentication lives withint acceptance root folder I found it logic to add it here, but happy to discuss other approaches.